website/content/docs/concepts/seal.mdx

```diff
@@ -157,255 +142,247 @@ and their API equivalents in the [/sys/init](/vault/api-docs/system/init) endpoi
 inconvenience of the downtime is an acceptable trade-off.
 
-~> **NOTE**: A backup should be taken before starting seal migration in case
-something goes wrong.
+You should make sure to create a backup before you start the seal migration process, in case something goes wrong.
+
+The seal migration operation requires both the old and new seals to be available during the migration. For example, if you migrate from auto unseal to Shamir seals, the service backing the auto unseal must be available during the migration.
 
-~> **NOTE**: Seal migration operation will require both old and new seals to be
-available during the migration. For example, migration from auto unseal to Shamir
-seal will require that the service backing the auto unseal is accessible during
-the migration.
+<Note>
 
-~> **NOTE**: Seal migration from auto unseal to auto unseal of the same type is
+Seal migration from auto unseal to auto unseal of the same type is
 supported since Vault 1.6.0. However, there is a current limitation that
-prevents migrating from AWSKMS to AWSKMS; all other seal migrations of the same
+prevents migrating from AWSKMS to AWSKMS. All other seal migrations of the same
 type are supported. Seal migration from one auto unseal type (AWS KMS) to
 different auto unseal type (HSM, Azure KMS, etc.) is also supported on older
-versions as well.
+versions.
+
+</Note>
 
 ### Migration post Vault 1.16.0 via Seal HA for Auto Seals (Enterprise)
 
-With Seal HA, migration between auto-unseal types (not including any Shamir
-seals) can be done fully online using Seal High Availability (Seal HA) without
-any downtime.
+You can perform an online migration between most auto-unseal types using seal high availability (seal HA) without any downtime.
+Seal HA does not support migrating Shamir seals.
 
-1. Edit the Vault configuration, and add the new, target seal configuration.
-1. Send the Vault process the SIGHUP signal, triggering a configuration reload.
+1. Edit the Vault configuration, and add the new target seal configuration.
+1. Send the Vault process the SIGHUP signal to trigger a configuration reload.
 1. Monitor the [`sys/sealwrap/rewrap`](/vault/api-docs/system/sealwrap-rewrap) endpoints,
-to see that rewrap is running, and/or [`sys/seal-backend-status`](/vault/api-docs/system/seal-backend-status),
+to confirm that rewrap is running, and/or [`sys/seal-backend-status`](/vault/api-docs/system/seal-backend-status),
 endpoints, waiting for `fully_wrapped` to be true, indicating all seal wrapped values are now
 wrapped by the new seal.  The logs also contain information about the rewrap progress.
-1. Edit the Vault configuration, removing the old seal configuration.
-1. Send the Vault process the SIGHUP signal, again allowing re-wrapping to complete.
+1. Edit the Vault configuration, and remove the old seal configuration.
+1. Send the Vault process the SIGHUP signal, again allowing rewrapping to complete.
```

Comment:
Changes present progressive tense to simple present. Remove hyphen from rewrap for consistency with other instances.